ui/src/components/Engine.jsx

```diff
@@ -217,6 +218,17 @@ class Engine extends React.Component {
       this.setState({
         consoleOpen: !this.state.consoleOpen,
       });
+      if(this.state.consoleOpen){
+        this.resizable.updateSize({ height: 0 });
+        this.setState({
+          minHeight: 0,
```

Comment:
`minHeight` sounds like a property of rendering, not a property of the engine.

Therefore the state tracked should probably not be anything to do with the "min height" -- that decision should be made by the `render` function based on whether `consoleOpen` or not.

Reply:
8f33bc7